components/utils/analytics.js

```diff
@@ -0,0 +1,10 @@
+import ReactGA from 'react-ga';
+
+export const initGA = () => {
+  ReactGA.initialize('UA-113130296-3');
```

Comment:
This is the FOOD-E ID, the correct ID is `UA-113130296-2`

Reply:
Updated.... https://github.com/ethicodeorg/website/commit/dcf8cea221b6212ac44c0bbf02380925d44374d2